packages/core/test/tag-input/tagInputTests.tsx

```diff
@@ -356,6 +362,13 @@ describe("<TagInput>", () => {
             pressEnterInInput(wrapper, NEW_VALUE);
             assert.strictEqual(wrapper.state().inputValue, "");
         });
+
+        it("does not clear the input if the input is controlled", () => {
+            const onChange = sinon.stub();
+            const wrapper = shallow(<TagInput onChange={onChange} values={VALUES} inputValue={NEW_VALUE} />);
```

Comment:
i'll bet if you did `inputValue=""` (controlled empty value) this test would fail.

Reply:
I don't think it would fail in this particular case, but I in general agree that we should be more explicit in checking if this prop is controlled.